### PR TITLE
feat: social sharing of groups and landscapes

### LINKED ISSUE #353: Share groups and landscapes via social
## Description
Add the ability to invite people to groups and landscapes via email, WhatsApp and Facebook.

## Additional context
image
image

### PR DESCRIPTION
**Pull request by paulschreiber**
## Description
Adds a share button to group and landscape pages. When clicked, a modal appears, allowing the user to share via email, WhatsApp and Facebook and to copy the link to the clipboard.

### Checklist
- [x] Corresponding issue has been opened
- [x] New tests added
- [X] English strings reviewed and copyedited
- [x] Strings are localized
- [X] Verified on mobile
- [X] Verified on desktop
- [x] Verified accessibility ([Accessibility - Web applications](https://docs.google.com/document/d/1VwPDyLqw7r_iISMgDZr1FdAmAqqOIIVIxQqsYA-HaHE/edit?usp=sharing))

### Related Issues
Fixes #353.

### CODE REVIEW

File: src/common/socialShare.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    window.open('https://wa.me/?text=' + pageUrl);
+  };
+
+  const shareViaFacebook = () => {
+    window.open('http://www.facebook.com/share.php?u=' + pageUrl);
+  };
+
+  const copyToClipboard = () => {
+    navigator.clipboard.writeText(pageUrl);
+    setButtonCopied(true);
```
Comment (cacarrara) [inline src/common/socialShare.js:46]: The status of link copied should be cleared after the modal is closed.

File: src/common/socialShare.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+            justifyContent="space-between"
+          >
+            <Button
+              variant="outlined"
+              startIcon={<EmailIcon sx={{ paddingRight: '5px' }} />}
+              onClick={shareViaEmail}
+            >
+              Email
+            </Button>
+            <Button
```
Comment (cacarrara) [inline src/common/socialShare.js:93]: The responsiveness of this component is not working properly:
image

File: src/group/components/GroupView.js [hunk 3]
```diff
@@ -124,15 +137,7 @@ const GroupView = () => {
         }}
       >
         <PageHeader header={group.name} />
-        <Restricted permission="group.change" resource={group}>
```
Comment (cacarrara) [inline src/group/components/GroupView.js:127]: Isn't this removing the edit button in Group view? I cannot see the edit button on groups where I'm a manager. Only the share button.

**Review by cacarrara — CHANGES_REQUESTED** [review]

**Review by cacarrara — COMMENTED** [review]
WhatsApp sharing is not working for me on desktop: Firefox.

File: src/group/components/GroupView.js (hunk 3 above)
Comment (paulschreiber) [reply src/group/components/GroupView.js:127]: It was moved to the group about card, per Hana.

**Review by cacarrara — APPROVED** [review]

**Pull request merged**